shared/makefile

```diff
@@ -124,12 +58,11 @@ rSharpInstrSharpUX: $(OUT_DIR)/rSharpUX.so
 
 rSharpLib: rSharpLibComp
 	if [ -e symbols.rds ] ; then cp -u -p symbols.rds $(INSTDIR)/lib$(R_ARCH) ; fi
-	-rm -f $(INSTDIR)/lib/*.*
-	-cp -u -p $(CLR_FACADE_BINS) $(INSTDIR)/lib
+
 	-cp -u -p ./$(RUNTIMECONFIG).json $(INSTDIR)/lib/
 
 rSharpNugetRestore:
-	-$(DOTNET_CMD) restore
+	-$(NUGET_CMD) restore
```

Comment:
This is a bit of a pain, I'll open an issue for it. For the time being, we need to install nuget on Ubuntu to restore packages because we are using a file called ```packages.config``` to force restoring dependencies for native C++ projects. That is not supported by ```dotnet``` cli. 

Using ```nuget``` means we need to have mono installed on the build computer.